src/index.html

```diff
@@ -3,12 +3,41 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="nav">
+        <a class="moyo-logo" href="index.html"><img src="./images/logo.png" alt="Moyo_logo"></a>
+        <nav class="nav-container">
+          <ul class="nav-list">
+            <li class="nav-item-list">
+              <a class="nav-link item-1 is-active"
+                 href="index.html"
+                 data-qa="hover">Apple
+              </a>
```

Comment:
```suggestion
              <a
                class="nav-link item-1 is-active"
                href="index.html"
                data-qa="hover"
              >
                Apple
              </a>
```
From [code style rules](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/adf4cdcc-c227-4325-8749-2aa13cf2ba72)